website/docs/language/functions/contains.mdx

```diff
@@ -5,8 +5,10 @@ description: The contains function determines whether a list or set contains a g
 
 # `contains` Function
 
-`contains` determines whether a given list or set contains a given single value
-as one of its elements.
+`contains` determines whether the list, tuple, or set given in its first
+argument contains at least one element that is equal to the value in the second
+argument, using the same definition of equality as the == operator described in
```

Comment:
Since this is an operator, code font feels appropriate?
```suggestion
argument, using the same definition of equality as the `==` operator described in
```